lib/scalyr/common/client.rb

```diff
@@ -126,11 +125,29 @@ def initialize(logger, add_events_uri, compression_type, compression_level,
         :compression_type => @compression_type,
         :compression_level => @compression_level,
     }
+  end  # def initialize
 
-    @http = Net::HTTP::Persistent.new
+  def client_config
+    # TODO: Eventually expose some more of these as config options, though nothing here really needs tuning normally
+    # besides SSL
+    c = {
+      connect_timeout: 10,
+      socket_timeout: 10,
+      request_timeout: 60,
+      follow_redirects: true,
+      automatic_retries: 1,
+      retry_non_idempotent: false,
+      check_connection_timeout: 200,
+      pool_max: 50,
+      pool_max_per_route: 25,
```

Comment:
Perhaps we could already expose those via config options:

* connect_timeout
* socket_timeout
* request_socket_timeout
* pool_max
* pol_max_per_route

``pool_max_per_route`` is the one we may want to experiment with in the near future. Aka see if we get even more gain if we set this value to >= number of output workers.